frontend/compositions/dashboardHeader/header.module.css

```diff
@@ -0,0 +1,156 @@
+/* default - mobile styles */
+.wrapper {
+  width: 100%;
+  height: 6.94rem;
```

Comment:
If you don't mind implementing - we have new global sizing variables in `styles/variables.css`! 
It maps the `px` values to their appropriate `rem` (ie here you could use `var(--size112)` to map to `7rem`)

Reply:
Apologies for the late response - will convert all rem to global sizing variables!